data-science-pipelines-operator/README.md

```diff
@@ -0,0 +1,104 @@
+# Data Science Pipelines Operator
+
+The Data Science Pipelines Operator (DSPO) is an OpenShift Operator that is used to deploy single namespace scoped 
+Data Science Pipeline stacks onto individual OCP namespaces.
+
+# Table of Contents
+
+1. [Quickstart](#quickstart)
+   1. [Pre-requisites](#pre-requisites)
+   2. [Deploy the Operator via ODH](#deploy-the-operator-via-odh)
+4. [Cleanup ODH Installation](#cleanup-odh-installation)
+
+# Quickstart
+
+To get started you will first need to satisfy the following pre-requisites:
+
+## Pre-requisites
+1. An OpenShift cluster that is 4.9 or higher.
+2. You will need to be logged into this cluster as [cluster admin] via [oc client].
+3. The OpenShift Cluster must have OpenShift Pipelines 1.7.2 or higher installed. Instructions [here][OCP Pipelines Operator].
+4. The Open Data Hub operator needs to be installed. You can install it via [OperatorHub][installodh].
+
+## Deploy the Operator via ODH
+
+On a cluster with ODH installed, create a namespace where you would like to install DSPO: 
```

Comment:
```suggestion
On a cluster with RHODS installed, create a namespace where you would like to install DSPO: 
```

Reply:
fixed, removed all mentions of odh in general, lmk if this works